docs/stencil-docs/reference-docs/handlebars-helpers-reference.md

```diff
@@ -1510,6 +1510,32 @@ Parse data with JSONparse.
 {{/JSONparse}}
 <!-- output: bar -->
 ```
+### {{JSONparseSafe}}
+
+Parse data safely with JSONparseSafe. This helper is similar to the JSONparse helper but allows handling of incorrect JSON data. Returns `parsed data` when JSON is correct; otherwise, renders an `else` block.

```

Comment:
```suggestion
Parse data safely with JSONparseSafe. This helper is similar to the JSONparse helper, but allows handling of incorrect JSON data. Returns `parsed data` when JSON is correct; otherwise, renders an `else` block.

```